web/src/routes/index.ts

```diff
@@ -1,4 +1,4 @@
-import Map from '../components/Map'
+import Map from 'components/Map'
 import Home from '../Home'
```

Comment:
should we use aliased import here too?

Reply:
Yes. But we move `Home` in the next series of changes (from `/src/Home` to `/src/components/Home). I'd like to implement the aliased import then.